argoverse/evaluation/competition_util.py

```diff
@@ -121,6 +124,54 @@ def generate_tracking_zip(input_path: str, output_path: str, filename: str = "ar
     shutil.rmtree(dirpath)
 
 
+def generate_stereo_zip(data_path: str, output_path: str) -> None:
```

Comment:
```suggestion
def generate_stereo_zip(data_dir: Path, output_dir: Path) -> None:
```

Mismatch in docstrings

Reply:
Right, I've updated it.